benchmarks/BenchmarkServer/Hubs/EchoHub.cs

```diff
@@ -10,23 +10,34 @@ namespace BenchmarkServer.Hubs
 {
     public class EchoHub : Hub
     {
-        public async Task Echo(int duration)
+        public async Task Broadcast(int duration)
         {
+            var sent = 0;
             try
             {
                 var t = new CancellationTokenSource();
                 t.CancelAfter(TimeSpan.FromSeconds(duration));
                 while (!t.IsCancellationRequested && !Context.ConnectionAborted.IsCancellationRequested)
                 {
-                    await Clients.All.SendAsync("echo", DateTime.UtcNow);
+                    await Clients.All.SendAsync("send", DateTime.UtcNow);
+                    sent++;
```

Comment:
Do we worry about sending over int.MaxValue messages in stress tests? 🐴

Reply:
Not doing stress tests 😛 
That variable is mostly a sanity check so I can make sure things are behaving as expected

Reply:
We will be doing stress tests at some point, but we can deal with that when we get there.